ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml

```diff
@@ -1973,6 +2012,12 @@ CHWSC:
   - failed_supply_air_temperature_alarm
   - high_supply_air_temperature_alarm
   - low_supply_air_temperature_alarm
+  - water_leak_alarm
+  - water_leak_alarm_1
+  - water_leak_alarm_2
+  - water_leak_alarm_3
+  - average_zone_air_temperature_sensor
```

Comment:
We should probably move this to a different type / create a new abstract for it. Discuss w/ Trevor